teammapper-backend/src/map/entities/mmpMap.entity.ts

```diff
@@ -11,35 +11,36 @@ import { MmpNode } from './mmpNode.entity'
 @Entity()
 export class MmpMap {
   @PrimaryGeneratedColumn('uuid')
-  id: string
+  id: string;
 
-  @Column({ type: 'timestamptz', nullable: true, default: () => 'now()' })
-  lastModified: Date
+  @Column({ type: 'timestamptz', nullable: true, default: () => 'CURRENT_TIMESTAMP' })
+  lastModified: Date | null = new Date();
 
   @Column({ type: 'timestamptz', nullable: true })
-  lastAccessed: Date
+  lastAccessed: Date | null;
 
-  @Column({ nullable: true })
+  @Column({ type: 'uuid', nullable: true })
   @Generated('uuid')
-  adminId: string
+  adminId: string | null;
 
-  @Column({ nullable: true, default: null })
+  @Column({ type: 'uuid', nullable: true, default: null })
   @Generated('uuid')
-  modificationSecret: string
+  modificationSecret: string | null = null;
```

Comment:
same here, why do we need to set it to null if the default is null?

Reply:
Isn't the default `undefined` if a variable is not set?

Reply:
depends on how typeorm handles it, you can simply test it yourself. If it's undefined, why not use undefined in the type? You added the type null, we could also use undefined. The parameter in the typeorm annotation `nullable` refers to the database, where `NULL` is the default. So we don't have to use null, we can use undefined. Your pick. But I wouldn't force it to null instead of undefined.